### PR TITLE
Leases view

### PR DESCRIPTION
**Pull request by dex4er**
This is a view for Leases:

image

image

### CODE REVIEW

File: src/common/front-end-routing/routes/cluster/config/leases/leases-route.injectable.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import isAllowedResourceInjectable from "../../../../../utils/is-allowed-resource.injectable";
+import { frontEndRouteInjectionToken } from "../../../../front-end-route-injection-token";
+
+const priorityClassesRouteInjectable = getInjectable({
+  id: "leases",
```
Comment (Nokel81) [inline src/common/front-end-routing/routes/cluster/config/leases/leases-route.injectable.ts:10]: 1. This variable name probably should be something like "leasesRouteInjectable"
2. This ID (which has to be globally unique) is a bit too generic, could you change it to be "leases-route"?

File: src/common/front-end-routing/routes/cluster/config/leases/navigate-to-leases.injectable.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import leasesRouteInjectable from "./leases-route.injectable";
+import { navigateToRouteInjectionToken } from "../../../../navigate-to-route-injection-token";
+
+const navigateToPriorityClassesInjectable = getInjectable({
```
Comment (Nokel81) [inline src/common/front-end-routing/routes/cluster/config/leases/navigate-to-leases.injectable.ts]: Same with this variable name

**Review by Nokel81 — COMMENTED** [review]
Two minor things and then this LGTM!

File: src/common/front-end-routing/routes/cluster/config/leases/leases-route.injectable.ts (hunk 1 above)
Comment (dex4er) [reply src/common/front-end-routing/routes/cluster/config/leases/leases-route.injectable.ts:10]: Fixed

File: src/common/front-end-routing/routes/cluster/config/leases/navigate-to-leases.injectable.ts (hunk 2 above)
Comment (dex4er) [reply src/common/front-end-routing/routes/cluster/config/leases/navigate-to-leases.injectable.ts]: Fixed

**Comment by dex4er** [comment]
BTW, I'm not sure if this view fits to Config menu but there is nothing better...

**Comment by Nokel81** [comment]
> BTW, I'm not sure if this view fits to Config menu but there is nothing better...

No I think it being part of Config makes sense

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**